### PR TITLE
Remove numbering in --list* commands

### PR DESCRIPTION
**Pull request by data-man**

### CODE REVIEW

**Comment by lerno** [comment]
Is this needed?

**Comment by data-man** [comment]
Why not? When updating syntax highlighting, additional actions are required.

**Comment by lerno** [comment]
I think better we remove the numbering completely for all that are numbered, I'll accept such a PR.

**Comment by data-man** [comment]
> we remove the numbering completely

Done.

File: src/compiler/compiler.c [hunk 1]
```diff
@@ -740,33 +740,32 @@ void print_syntax(BuildOptions *options)
...
-			printf("%2d %s\n", i + 1, attribute_list[i]);
+			printf("%s\n", attribute_list[i]);
 		}
 	}
 	if (options->print_builtins)
 	{
 		for (int i = 0; i < NUMBER_OF_BUILTINS; i++)
 		{
-			printf("%3d $$%s\n", i + 1, builtin_list[i]);
+			printf("%s\n", builtin_list[i]);
```
Comment (lerno) [inline src/compiler/compiler.c:757]: You removed too much here and also below.

File: src/compiler/compiler.c [hunk 2]
```diff
@@ -740,33 +740,32 @@ void print_syntax(BuildOptions *options)
...
 		}
-		puts("---");
 		for (int i = 0; i < NUMBER_OF_BUILTIN_DEFINES; i++)
 		{
-			printf("%2d $$%s\n", i + 1, builtin_defines[i]);
+			printf("%s\n", builtin_defines[i]);
```
Comment (lerno) [inline src/compiler/compiler.c:761]: $$ is missing.

File: src/compiler/compiler.c [hunk 3]
```diff
@@ -740,33 +740,32 @@ void print_syntax(BuildOptions *options)
...
 		}
 	}
 	if (options->print_type_properties)
 	{
 		for (int i = 0; i < NUMBER_OF_TYPE_PROPERTIES; i++)
 		{
-			printf("%2d .%s\n", i + 1, type_property_list[i]);
+			printf("%s\n", type_property_list[i]);
```
Comment (lerno) [inline src/compiler/compiler.c:768]: . is missing.

Comment (data-man) [reply src/compiler/compiler.c:768]: Because dot isn't a part of name.

**Comment by lerno** [comment]
Can we put this in 0.6.0 (so dev) rather than 0.5.5 (master)?

**Pull request closed without merging**